openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -42,7 +42,7 @@ function download_serving {
     target_file="$target_dir/$index-$file"
 
     if [[ ${KNATIVE_SERVING_MANIFESTS_DIR} = "" ]]; then
-      url="https://raw.githubusercontent.com/openshift/serving/${branch}/openshift/release/artifacts/$index-$file"
+      url="https://raw.githubusercontent.com/openshift/knative-serving/${branch}/openshift/release/artifacts/$index-$file"
```

Comment:
Url still works due to the github redirects I suupose that is [kept for ever.](https://github.community/t/how-long-does-github-forward-renamed-moved-repos/582/5)
Both  ```wget https://raw.githubusercontent.com/openshift/serving/release-v1.3/openshift/release/artifacts/1-serving-crds.yaml
wget https://raw.githubusercontent.com/openshift/knative-serving/release-v1.3/openshift/release/artifacts/1-serving-crds.yaml
```
return the file. But I agree let's update it.